Build fails in Debian 11 due to dh-systemd package dependency

The functionality of dh-systemd has been merged into the debhelper-package in Debian 11 (even earlier as it seems, but since Debian 11 the former has been removed completely). Because of this I get a `E: Unable to locate package dh-systemd` while following https://wiki.trezor.io/User_manual:Running_a_local_instance_of_Trezor_Wallet_backend_(Blockbook) (step 4).

I have removed `dh-systemd` from 
build/templates/blockbook/debian/control ,
build/templates/backend/debian/control and
build/docker/deb/Dockerfile
and now it builds just fine!

I was able to reproduce this issue on Ubuntu 22.04 (Jammy) beta:
```
 ---> Running in d7639717a32d
Cloning into 'blockbook'...
Removing intermediate container d7639717a32d
 ---> 9b3b0ddf1b70
Step 23/25 : ADD Makefile /build/Makefile
 ---> ca888ec5ed1e
Step 24/25 : VOLUME /out
 ---> Running in 6dce4d1e59ba
Removing intermediate container 6dce4d1e59ba
 ---> 96de6122979f
Step 25/25 : WORKDIR /build
 ---> Running in 63c6e54346f8
Removing intermediate container 63c6e54346f8
 ---> 0b459ee6b0a8
Successfully built 0b459ee6b0a8
Successfully tagged blockbook-build:latest
Building image blockbook-build-deb...
Sending build context to Docker daemon   59.9kB
Step 1/6 : FROM blockbook-build:latest
 ---> 0b459ee6b0a8
Step 2/6 : RUN apt-get update &&     apt-get upgrade -y &&     apt-get install -y devscripts debhelper make dh-systemd dh-exec &&     apt-get clean
 ---> Running in d8559b60bf08
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package dh-systemd
The command '/bin/sh -c apt-get update &&     apt-get upgrade -y &&     apt-get install -y devscripts debhelper make dh-systemd dh-exec &&     apt-get clean' returned a non-zero code: 100
make[1]: *** [Makefile:55: .deb-image] Error 100
make[1]: Leaving directory '/home/user/blockbook'
make: *** [Makefile:44: build-images] Error 2
```
Even on Ubuntu 20.04, `dh-systemd` is already a transitional package without contents:
```
dh-systemd - debhelper add-on to handle systemd unit files - transitional package
```
Removing mentions of `dh-systemd`, as proposed, makes the build succeed.

--- a/build/docker/deb/Dockerfile
+++ b/build/docker/deb/Dockerfile
@@ -4,7 +4,7 @@ FROM blockbook-build:latest
 
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y devscripts debhelper make dh-systemd dh-exec && \
+    apt-get install -y devscripts debhelper make dh-exec && \
     apt-get clean
 
 ADD gpg-keys /tmp/gpg-keys

--- a/build/templates/backend/debian/control
+++ b/build/templates/backend/debian/control
@@ -3,7 +3,7 @@ Source: backend
 Section: satoshilabs
 Priority: optional
 Maintainer: {{.Meta.PackageMaintainerEmail}}
-Build-Depends: debhelper, wget, tar, gzip, make, dh-systemd, dh-exec
+Build-Depends: debhelper, wget, tar, gzip, make, dh-exec
 Standards-Version: 3.9.5
 
 Package: {{.Backend.PackageName}}

--- a/build/templates/blockbook/debian/control
+++ b/build/templates/blockbook/debian/control
@@ -3,7 +3,7 @@ Source: blockbook
 Section: satoshilabs
 Priority: optional
 Maintainer: {{.Meta.PackageMaintainerEmail}}
-Build-Depends: debhelper, dh-systemd, dh-exec
+Build-Depends: debhelper, dh-exec
 Standards-Version: 3.9.5
 
 Package: {{.Blockbook.PackageName}}

